### ISSUE TITLE
Image build fails

### ISSUE THREAD

**Issue by dannyim** [PolarGeospatialCenter/gdal-docker#1]
I am attempting to build the image and it is failing on the conda install command.

Here is an excerpt from the build log where it fails:
```
Step 18/25 : RUN conda install --yes pip &&     conda clean --all --yes
 ---> Running in ef118aa0a9c8
Solving environment: ...working... done


==> WARNING: A newer version of conda exists. <==
  current version: 4.5.11
  latest version: 4.8.1

Please update conda by running

    $ conda update -n base -c defaults conda


ca-certificates-2019 | 132 KB    | ########## | 100%
wheel-0.33.6         | 35 KB     | ########## | 100%
libedit-3.1.20181209 | 188 KB    | ########## | 100%
zlib-1.2.11          | 120 KB    | ########## | 100%
xz-5.2.4             | 366 KB    | ########## | 100%
certifi-2019.11.28   | 156 KB    | ########## | 100%
pip-19.3.1           | 1.9 MB    | ########## | 100%
_libgcc_mutex-0.1    | 3 KB      | ########## | 100%
python-3.8.1         | 57.8 MB   | ########## | 100%
setuptools-44.0.0    | 652 KB    | ########## | 100%
ld_impl_linux-64-2.3 | 645 KB    | ########## | 100%
libgcc-ng-9.1.0      | 8.1 MB    | ########## | 100%
openssl-1.1.1d       | 3.7 MB    | ########## | 100%
ncurses-6.1          | 958 KB    | ########## | 100%
sqlite-3.30.1        | 1.9 MB    | ########## | 100%

## Package Plan ##

  environment location: /opt/conda

  added / updated specs:
    - pip


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    ca-certificates-2019.11.27 |                0         132 KB
    wheel-0.33.6               |           py38_0          35 KB
    libedit-3.1.20181209       |       hc058e9b_0         188 KB
    zlib-1.2.11                |       h7b6447c_3         120 KB
    xz-5.2.4                   |       h14c3975_4         366 KB
    certifi-2019.11.28         |           py38_0         156 KB
    pip-19.3.1                 |           py38_0         1.9 MB
    _libgcc_mutex-0.1          |             main           3 KB
    python-3.8.1               |       h0371630_1        57.8 MB
    setuptools-44.0.0          |           py38_0         652 KB
    ld_impl_linux-64-2.33.1    |       h53a641e_7         645 KB
    libgcc-ng-9.1.0            |       hdf63c60_0         8.1 MB
    openssl-1.1.1d             |       h7b6447c_3         3.7 MB
    ncurses-6.1                |       he6710b0_1         958 KB
    sqlite-3.30.1              |       h7b6447c_0         1.9 MB
    ------------------------------------------------------------
                                           Total:        76.6 MB

The following NEW packages will be INSTALLED:

    _libgcc_mutex:    0.1-main
    ld_impl_linux-64: 2.33.1-h53a641e_7
    xz:               5.2.4-h14c3975_4

The following packages will be UPDATED:

    ca-certificates:  2018.03.07-0            --> 2019.11.27-0
    certifi:          2018.8.24-py27_1        --> 2019.11.28-py38_0
    libedit:          3.1.20170329-h6b74fdf_2 --> 3.1.20181209-hc058e9b_0
    libgcc-ng:        8.2.0-hdf63c60_1        --> 9.1.0-hdf63c60_0
    ncurses:          6.1-hf484d3e_0          --> 6.1-he6710b0_1
    openssl:          1.0.2p-h14c3975_0       --> 1.1.1d-h7b6447c_3
    pip:              10.0.1-py27_0           --> 19.3.1-py38_0
    python:           2.7.15-h1571d57_0       --> 3.8.1-h0371630_1
    setuptools:       40.2.0-py27_0           --> 44.0.0-py38_0
    sqlite:           3.24.0-h84994c4_0       --> 3.30.1-h7b6447c_0
    wheel:            0.31.1-py27_0           --> 0.33.6-py38_0
    zlib:             1.2.11-ha838bed_2       --> 1.2.11-h7b6447c_3


Downloading and Extracting Packages
Preparing transaction: ...working... done
Verifying transaction: ...working... done
Executing transaction: ...working... done
Traceback (most recent call last):
  File "/opt/conda/bin/conda", line 7, in <module>
    from conda.cli import main
ModuleNotFoundError: No module named 'conda'
The command '/bin/sh -c conda install --yes pip &&     conda clean --all --yes' returned a non-zero code: 1
```

**Comment by dannyim** [comment]
It looks like conda is updating the python installation to python-3.8.1, even though the image is based on python 2.  I was able to get the build working by updating the parent image to continuumio/miniconda2:4.7.12

**Comment by dannyim** [comment]
Fixed in #3
